addons/website/views/snippets/s_numbers_charts.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_numbers_charts" name="Numbers Charts">
```

Comment:
Shouldn't this be named `s_numbers_chart` and `Numbers Chart` ? (If you change it, be sure to also rename the file name)

Reply:
There are three charts, so I guess it makes sense to use plural for both ?

Reply:
Oh... ok the progress bars count as charts. - Somehow I saw them as thick separators.